src/oSIP.Net.Tests/AcceptEncodingHeaderTests.cs

```diff
@@ -8,37 +8,32 @@ public class AcceptEncodingHeaderTests
         [Test]
         public void Shall_stringify_header()
         {
-            using (var header = new AcceptEncodingHeader())
-            {
-                header.Element = "gzip";
-                header.Parameters.Add(new GenericParameter("foo", "bar"));
-                Assert.That(header.ToString(), Is.EqualTo("gzip;foo=bar"));
-            }
+            var header = new AcceptEncodingHeader();
+            header.Element = "gzip";
+            header.Parameters.Add(new GenericParameter("foo", "bar"));
+            Assert.That(header.ToString(), Is.EqualTo("gzip;foo=bar"));
         }
 
         [Test]
         public void Shall_parse_header()
         {
-            using (AcceptEncodingHeader header = AcceptEncodingHeader.Parse("gzip;foo=bar"))
-            {
-                Assert.That(header.Element, Is.EqualTo("gzip"));
-                Assert.That(header.Parameters[0].Name, Is.EqualTo("foo"));
-                Assert.That(header.Parameters[0].Value, Is.EqualTo("bar"));
-            }
+            AcceptEncodingHeader header = AcceptEncodingHeader.Parse("gzip;foo=bar");

```

Comment:
Use TryParse with Assert.That in the parse tests, and Parse in the clone tests. That way, both Parse and TryParse are used in tests.